lib/cdo/poste.rb

```diff
@@ -217,7 +207,6 @@ def send(delivery)
       params.merge(
         {
           recipient: OpenStruct.new(recipient),
-          encrypted_id: encrypted_id,
           unsubscribe_link: unsubscribe_url,
           tracking_pixel: poste_url("/o/#{encrypted_id}"),
```

Comment:
Note, we still use the `encrypted_id` in both the `unsubscribe_link` and `tracking_pixel` which end up getting passed through to the template as locals.